docs/src/basics/exporters.md

```diff
@@ -0,0 +1,89 @@
+# Exporters
+
+As Zeebe processes jobs and workflows, or performs internal maintenance (e.g.
+raft failover), it will generate an ordered stream of records:
+
+![record-stream](/basics/exporters-stream.png)
+
+While the clients provide no way to inspect this stream directly, Zeebe can load
+and configure user code that can process each and every one of those records, in the form of an exporter.
+
+An **exporter** provides a single entry point to process every record that is
+written on a stream.
+
+With it, you can:
+
+* Persist historical data by pushing it to an external data warehouse
+* Export records to a visualization tool (e.g. [zeebe-simple-monitor](https://github.com/zeebe-io/zeebe-simple-monitor/))
+
+You can find a reference exporter implementation in the form of the Zeebe-maintained
+[ElasticSearch exporter](https://github.com/zeebe-io/zeebe/tree/{{commit}}/exporters/elasticsearch-exporter).
+
+For more information, you can read the [reference information page](reference/exporters.html).
+
+> **Note**: it is not recommended to use exporters for orchestration purposes, as
+> that is what workflows are for.
+
+## Usage
```

Comment:
I think this section fits better into the Reference -> Exporters chapter. The Basics section is meant to explain the principles of Zeebe to people (maybe non-technical) who are new to it. As a user who has understood these principles and who wants to build something with Zeebe, I should not have to go back to the Basics section.